serde/de.py

```diff
@@ -7,10 +7,17 @@
 import collections
 import dataclasses
 import functools
+import sys
 import typing
 from dataclasses import dataclass, is_dataclass
 from typing import Any, Callable, Dict, List, Optional, TypeVar
 
+if sys.version_info[1] < 8:
+    # get_args was added in python 3.8
+    from typing_utils import get_args
+else:
+    from typing import get_args
```

Comment:
Can we use [`compat.get_args`](https://github.com/yukinarit/pyserde/blob/648471c85398d33a4d8627462de4db17094ef639/serde/compat.py#L130-L137) instead? 🤔 

I don't really know the differences between typing_extensions and typing_utils though

Reply:
Thanks, didn't know about this! Updated.